src/test/resources/appointment-data5.json

```diff
@@ -36,7 +35,21 @@
       "locality" : "Cardiff"
     }
   },
-  "delta_at" : "20200623121303006153",
+  "sensitive_data" : {
```

Comment:
appointed_before on line 27 should be an ISODate, we expect it to come in as an Instant in the db model.

Reply:
done

Reply:
because I have changed this, the OfficerAppointmentsRepositoryITest doesn't work anymore. Probably better to be fixed in John's ticket DSND-1679, fix is a little trickier than first thought and would cause merge conflicts probably.